emoji/src/main/java/com/vanniktech/emoji/EmojiSpan.java

```diff
@@ -3,8 +3,11 @@
 import android.content.Context;
 import android.graphics.Canvas;
 import android.graphics.Paint;
+import android.graphics.drawable.AnimationDrawable;
 import android.graphics.drawable.Drawable;
 import android.text.style.DynamicDrawableSpan;
+import android.util.Log;
+
 import com.vanniktech.emoji.emoji.Emoji;
 
 final class EmojiSpan extends DynamicDrawableSpan {
```

Comment:
Instead of extending this span, we want a special `Span` implementation for animations I think. Drawing the animation ourselfs on the `Canvas` is hard.